packages/admin-panel/src/VizBuilderApp/views/Main.js

```diff
@@ -21,9 +26,50 @@ const RightCol = styled(FlexColumn)`
   flex: 1;
 `;
 
+const StyledAlert = styled(SmallAlert)`
+  margin-top: 30px;
+  margin: auto;
+`;
+
+const Progress = styled(CircularProgress)`
+  margin-top: 30px;
+  margin: auto;
```

Comment:
Same here? Do you mean: `margin: 30px auto auto;`?

Reply:
Sorry, it was a mistake, the 30px is redundant